examples/video-chat/ios/Podfile

```diff
@@ -49,8 +49,10 @@ target 'FishjamExample' do
       :mac_catalyst_enabled => false
     )
   end
-  pod 'FishjamClient', :path => '../../../packages/ios-client/'
+#   pod 'FishjamCloudClient'
+  pod 'FishjamCloudClient', :path => '../../../'
```

Comment:
why this change, and why leftover comment?

Reply:
ios was released before, so new name is necessary.

Reply:
and the path is correct?